src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/msquic_generated_macos.cs

```diff
@@ -5,6 +5,8 @@
 //
 #pragma warning restore IDE0073
 
+using System.Runtime.InteropServices;
```

Comment:
What is this using added for? Its not on our side. I can add it if necessary, but I'd like to keep these as identical as possible.

Reply:
https://github.com/microsoft/msquic/blob/8cba1150674cbecda7a136f344acc64c65f49932/src/cs/lib/msquic_generated_macos.cs#L8

It's on your side. Or am I going crazy? :smile:

Reply:
Oh I was looking at the windows file... So I'm going crazy. Let me remove it, theres no reason for it to be there.

Reply:
Updated.